bin/server.js

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env node
+var debug             = require('debug')('github:server');
+var base              = require('taskcluster-base');
+var api               = require('../lib/api');
+var path              = require('path');
+var Promise           = require('promise');
+var exchanges         = require('../lib/exchanges');
+var _                 = require('lodash');
+
+/** Launch server */
+var launch = async function(profile, publisher) {
```

Comment:
Agree 100% about making it nice, and I can understand the reasoning behind relying heavily on end-to-end tests to increase velocity: integration tests pack a lot of bang for their buck.  

Having a dummy publisher and stats drain would be awesome to have for unit tests; it's cumbersome to need credentials just to get started running tests. It also precludes contributors who don't have access to all of our services, and makes offline development more difficult/impossible.